redirects.json

```diff
@@ -2869,13 +2851,13 @@
   },
   {
     "source":"/lib-v1/restapi/existing-resources/q/platform/js/",
-    "target":"/javascript/prev/build-a-backend/restapi/existing-resources/",
+    "target":"/javascript/build-a-backend/restapi/existing-resources/",
```

Comment:
[`/lib-v1`](https://docs.amplify.aws/lib-v1/restapi/existing-resources/q/platform/js/) is v6 content. redirected to the v6 content (no page at `/prev/`)